modules/internal/ExportWrappers.chpl

```diff
@@ -24,6 +24,31 @@ module ExportWrappers {
   use CPtr;
   use SysCTypes;
 
+  private proc _initDynamicEndCount() {
+    var endCount = _endCountAlloc(false);
+    __primitive('set dynamic end count', endCount);
```

Comment:
I'm leaning towards using `chpl_task_setDynamicEndCount(endCount)` instead of the primitive. Gets us a little type-checking and seems nice to avoid more primitives (even though they map to the same thing)